app/models/decidim/time_tracker/activity.rb

```diff
@@ -100,7 +101,7 @@ def answered_by?(user)
 
       # used as a unique idenfier when answering the task associated questionnaire
       def session_token(user)
-        "#{user.id}-#{id}"
+        "#{Digest::SHA1.hexdigest(user.id.to_s)}-#{id}"
```

Comment:
I don't know about this, is pretty easy to disclose the user.id anyway, you just need to loop integers to do it.
I we want to do this, we should use this: https://github.com/decidim/decidim/pull/6850

Not available in 0.22 yet though

Reply:
actually, not available until release 0.24 probably

Reply:
For the moment, I think we can simply not export or show this information anywhere public